src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1174,6 +1177,42 @@ public JSONObject getDebugState(String conferenceId, String endpointId)
         return debugState;
     }
 
+    /**
+     * Gets statistics for the different {@code PacketQueue}s that this bridge
+     * uses.
+     * TODO: is there a better place for this?
+     */
+    public JSONObject getQueueStats()
+    {
```

Comment:
since the string creation can be costly i'm tempted to say we should always do constants for any un-changing strings like this just as a practice.  what do you think?

Reply:
Okay, will fix

Reply:
Looking at it again, I think this code is better without constants. It's the only place where these strings are used, and having if they are defined in place everything is in the method body and easier to understand. As for performance, this doesn't create new string instances. The compiler "interns" literals, so it should be equivalent to defining a static final object in terms of performance.